lib/buttons/ButtonHelp.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { InfoSVG } from "../svg/Info";
+import { valEmpty } from "../utils";
+
+export type ButtonHelpProps = {
+  onClick?: () => void;
+  light?: boolean;
+};
+
+/*
+ * Help button used throughout dashboard apps.
+ */
+export const ButtonHelp = ({ onClick, light }: ButtonHelpProps) => (
+  <button
```

Comment:
add the style prop and `style={style}` in case users want to extend it via plan CSS.

We will most likely support this throughout the component library.